packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -401,6 +401,9 @@ export class TeamsBotImpl {
           `Before running this bot, please manually update bot's message endpoint(${this.config.provision.siteEndpoint}${CommonStrings.MESSAGE_ENDPOINT_SUFFIX}). Click 'Get Help' button for more details.`,
           Links.UPDATE_MESSAGE_ENDPOINT
         );
+        Logger.info(
+          `Please manually update bot's message endpoint(${this.config.provision.siteEndpoint}${CommonStrings.MESSAGE_ENDPOINT_SUFFIX}).`
```

Comment:
Since this sentence is the same as above, I'd like to have a variable to for it to remove duplicate code.

Reply:
sure.

Reply:
updated.